PC/bdist_wininst/install.c

```diff
@@ -2553,8 +2565,10 @@ int DoUninstall(int argc, char **argv)
             lines_buffer_size += 10;
             lines = (char **)realloc(lines,
                                      sizeof(char *) * lines_buffer_size);
-            if (!lines)
+            if (!lines) {
```

Comment:
The memory pointed by the old value of `lines` is leaked if `realloc()` returns NULL.

And seems `lines` is leaked in the code below.